nodeadm/test/e2e/cases/kubelet-flags/config.yaml

```diff
@@ -14,3 +14,6 @@ spec:
       - --v=5
       - --node-labels=foo=bar,foo2=baz
       - --register-with-taints=foo=bar:NoSchedule
+      - |
+        --foo
+        --bar
```

Comment:
Not sure we want to allow this, looks pretty odd. What's the use case?

Reply:
included this for edge case coverage, not as an intended use case

Reply:
The `EnvironmentFile` directive already has a bunch of documented handling of newlines, escape characters, etc. so I don't think we should be doing `santitize`, the underlying issue here was us wrapping the user's input in `"`, right? Isn't removing that sufficient?

> a "'"-quoted value after the "=" can span multiple lines and contain any character verbatim other than single quote
https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html

Reply:
got it, yea won't do any modification to the user's input. just going to fix the wrapped content issue